### ISSUE TITLE
`evaluate_script` throws an exception

### ISSUE THREAD

**Issue by arthwood** [thoughtbot/capybara-webkit#1028]
`capybara-webkit` version: `v1.14.0` (works fine on `v1.13.0`)
`capybara` version: any but I run it on `2.13.0`

I've got helper method:

```ruby
    def get_style(node)
      path = node.path

      result = page.evaluate_script <<-EOS.squish
        var node = document.evaluate("#{path}", document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue;

        window.getComputedStyle(node);
      EOS

      pp page.driver.console_messages

      result
    end
```

output from console shows:

```javascript
[{:line_number=>499,
  :message=>
   "NoModificationAllowedError: DOM Exception 7: An attempt was made to modify an object where modifications are not allowed.",
  :source=>"undefined"}]
```

I believe it's due to [this](https://github.com/thoughtbot/capybara-webkit/compare/v1.13.0...v1.14.0#diff-16abd16de283b3e3b50eea0cede396f3R27) change.

When I simulate wrapping result in real app in the browser the error is more descriptive (the `_result` passed to wrapper is a `CSSStyleDeclaration` in my case):

> Uncaught DOMException: Failed to set the 'align-content' property on 'CSSStyleDeclaration': These styles are computed, and therefore the 'align-content' property is read-only.

**Comment by twalpole** [comment]
@arthwood  Could you try PR #1032 and see if that fixes the issue for you?

**Comment by arthwood** [comment]
@twalpole Yup, that ref works fine. 👍

**Comment by twalpole** [comment]
Fixed via PR #1032
